backend/environment.d.ts

```diff
@@ -0,0 +1,15 @@
+declare global {
```

Comment:
This is so env variables are recognized by Typescript. DM me if you are testing locally and need a `.env` copy